osu.Game/Rulesets/Scoring/ScoreProcessor.cs

```diff
@@ -417,7 +417,7 @@ public virtual void PopulateScore(ScoreInfo score)
                 score.MaximumStatistics[result] = maximumResultCounts.GetValueOrDefault(result);
 
             // Populate total score after everything else.
-            score.TotalScore = (long)Math.Round(ComputeScore(ScoringMode.Standardised, score));
+            score.TotalScore = ComputeScore(ScoringMode.Standardised, score);
```

Comment:
I'm concerned about all these places using round, and the new code using truncation (floor). On osu-stable there's an issue where mania scores could get to 999999 with 100% acc that's with a jank conditional. I believe we still want rounding to avoid such an issue appearing again here.